testdata/v1beta1/clustertask/pipelines/s2i-dotnet.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: s2i-dotnet-pipeline
+spec:
+  params:
+  - name: EXAMPLE_REVISION
+    description: git branch of .NET example
+    default: dotnetcore-3.1
```

Comment:
This cannot be hard-coded, you need to choose correct branch for each version of .Net

Reply:
This is just the default value, in the step I am passing this value as well

Reply:
oh, my bad